redesign-project/src/styles.scss

```diff
@@ -0,0 +1,97 @@
+/* For desktop: */
+body {
+    background-color: #62718C
+}
+
+.background {
+    display: none;
+}
+
+.desktop_background {
+    width: 96%;
+    position: absolute;
```

Comment:
I definitely think you can achieve the layout you're going for with less (or no) absolute positioning.